src/main/java/reposense/parser/ArgsParser.java

```diff
@@ -92,6 +93,12 @@ private static ArgumentParser getArgumentParser() {
                         + "If not provided, default file formats will be used.\n"
                         + "Please refer to userguide for more information.");
 
+        parser.addArgument("-ignore")
+                .metavar("BOOLEAN")
+                .type(new BooleanArgumentType())
```

Comment:
As mentioned by @eugenepeh, probably would be better if we can just use a flag without having to explictly state `true` for this.
You can try this SO post [here](https://stackoverflow.com/questions/28813481/setting-a-flag-using-a-command-line-argument-with-argparse4j) on how to do this.

Reply:
Have edited it.